babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -991,31 +991,35 @@ SqlNode CurrentDateFunction() :
 
 SqlNode DateTimeTerm() :
 {
-    final SqlNode e;
-    SqlIdentifier timeZoneValue;
+    final SqlNode dateTimePrimary;
+    final SqlNode displacement;
 }
 {
     (
```

Comment:
This is the outer parentheses I'm referring to that we could probably remove.

Reply:
The one that starts at 998 actually ends at 1004. The one from 1005 to 1025 are removed.

Reply:
Ah thanks for clarifying. I was looking at the wrong pair of parentheses then.